src/main/scala/com/lunatech/pi/Futures.scala

```diff
@@ -96,9 +106,6 @@ object Main {
         fjPool.shutdown()
     }
     //val pi: BigDecimal = Await.result(piF, Inf)
-
-
-
 //    val stopTime = System.currentTimeMillis
```

Comment:
I personally find commented out code really distracting. So if you need to leave these in here for illustration purposes, I'd recommend giving more context to why they are in here, or just remove them.

Reply:
Fully agree with that.